pkg/build/builder/docker_test.go

```diff
@@ -117,11 +117,11 @@ func cmpASTs(original *parser.Node, edited *parser.Node) int {
 		index++
 	}
 
-	originalChildren := make([]*parser.Node, 0)
+	var originalChildren []*parser.Node
```

Comment:
Even if this is a test, I think `originalChildren := make([]*parser.Node, len(original.Children))` is better as you avoid reallocs.